apps/keyboard/js/layouts/emoji.js

```diff
@@ -0,0 +1,458 @@
+Keyboards.emoji = {
+  label: 'Emoji',
+  types: ['text', 'url', 'email'],
+  menuLabel: 'Emoji',
+  pages: [{   // Page 0 - People
+    pageType: 'emoji',
+    sectionKeys: [ { compositeKey: '😄', type: 'emoji' }, { compositeKey: '😃', type: 'emoji' },
```

Comment:
@timdream, I thought this way it might be easier to change how we split the sections once we want to change the visual style, say from 6 x 3 for a section to 7 x 3.

Besides, for the landscape mode, we still don't know how to split these sections, so I would not want to keep the "sections" fixed in the layout definition file.

Thanks.